cocos/platform/CCApplication.h

```diff
@@ -190,7 +190,7 @@ class CC_DLL Application
     virtual void onCreateView(PixelFormat& pixelformat, DepthFormat& depthFormat, int& multisamplingCount);
     
 private:
-    void createView(const std::string& name, int width, int height);
+    void createView(const std::string& name, int& width, int& height, bool fullscreen);
```

Comment:
why use reference for int?

Reply:
This modification is tricky because the full screen is controlled by the bottom layer and the top layer needs to know the modified screen size, so it uses a reference to pass back the width and height of the full screen.

Reply:
感觉没有必要通过引用修改. 后续代码应该通过 `getSize` 方法获取 宽高.